src/stats/utils.js

```diff
@@ -57,3 +57,18 @@ exports.expectIsRepo = (err, res) => {
   expect(res.repoPath).to.be.a('string')
   expect(res.version).to.be.a('string')
 }
+
+exports.expectIsRepoHumanized = (err, res) => {
+  expect(err).to.not.exist()
+  expect(res).to.exist()
+  expect(res).to.have.a.property('numObjects')
+  expect(res).to.have.a.property('repoSize')
+  expect(res).to.have.a.property('repoPath')
+  expect(res).to.have.a.property('version')
+  expect(res).to.have.a.property('storageMax')
+  expect(res.numObjects).to.be.a('number')
+  expect(res.repoSize).to.eql('30.1 kB')
```

Comment:
This is a brittle assertion - it may work for now, but it could easily change in the future (the default assets in a repo could be updated for example) causing this test to fail.

We should instead assert that this value is in the format expected since an actual value is almost impossible to test for without mocking.

Reply:
Got it. I'll use a regex to test the expected format.